### ISSUE TITLE
Bakkesmod Linux support

### ISSUE THREAD

**Issue by TimestaticYT** [bakkesmodorg/BakkesModInjectorCpp#16]
I tried installing Bakkesmod on Linux Manjaro but since the [Wiki](https://bakkesmod.fandom.com/wiki/Running_in_Linux_/_Steam_Proton) article is outdated I tried the Windows 7 version of Bakkes. I used proton-ge-custom and `protontricks --gui`to install vcrun2017, vcrun2019, d3dcompiler_42.dll, d3dcompiler_43.dll, d3dcompiler_47.dll and corefonts since it was suggested by [this older Guide](https://www.reddit.com/r/RocketLeague/comments/hr5cb7/linuxguide_bakkesmod_on_linux/?utm_source=share&utm_medium=web2x&context=3). I then was able to launch the Windows 7 version but it said i should install the x64 and x86 VC Redist but I already installed vcrun2019 and 2017. I also installed the Windows 10 version of bakkesmod later on with protontricks by setting `protontricks 252950 winecfg` to windows 10 and after launching the windows 10 version it also said the VC Redist as it couldn't inject the DLL due to missing dependecy. In winetricks it said that vcrun2015 should be included in vcrun2019. As a sidenote I also tried to install both x86 visual C++ and x64 visual C++ manually through protontricks and it didn't help.

TL;DR It would be great if the Wiki page for Linux support got updated as I'm unable to install BakkesMod with my current Skill as I loved BakkesMod when I still used Windows 10

**Comment by sadn1ck** [comment]
+1, same issue, both on epic games version (via lutris) and steam version (protontricks). Would be great if we can have some sort of fix.

**Comment by TimestaticYT** [comment]
@sadn1ck was kinda disappointed since they even had a wiki page for the old version but i just couldn't get it working with the new one

**Comment by sadn1ck** [comment]
@TimestaticYT some of us actually got it working a few weeks back. 

[Here](https://github.com/bakkesmodorg/BakkesMod2-Plugins/issues/2#issuecomment-821990181)

**Comment by Jarmander333** [comment]
@Bakkes needs to think a little bit further into this due to the introduction of the Steam Deck.  If this were to happen, I'd be super thrilled!

**Comment by TimestaticYT** [comment]
Yeah Steam Deck support for RL would be huge

**Comment by Ciflire** [comment]
You might want to check [this](https://github.com/CrumblyLiquid/BakkesLinux), there are a few errors that I mentionned in an issue, but his installation script works pretty good besides that.
I have a working version at the momentusing his scripts
